doc/source/reference/arrays.dtypes.rst

```diff
@@ -178,7 +178,7 @@ Built-in Python types
     :class:`bool`     :class:`bool\_`
     :class:`float`    :class:`float\_`
     :class:`complex`  :class:`cfloat`
-    :class:`str`      :class:`string`
+    :class:`str`      :class:`bytes` (Python2) or :class:`unicode\_` (Python3)
```

Comment:
Shouldn't this be `bytes_`?

How about not listing `str` at all, and having
```
:class:`bytes`      :class:`bytes_`
```

Perhaps a note after the table that `str` in python is just an alias for either `bytes` or `unicode`.